charts/operator-wandb/templates/ingress.yaml

```
@@ -1,4 +1,18 @@
 {{- if .Values.ingress.issuer.create }}
+{{- if eq .Values.ingress.issuer.provider "google" }}
+---
+apiVersion: networking.gke.io/v1
+kind: ManagedCertificate
+metadata:
+  name: {{ .Release.Name }}-cert
+  namespace: {{ .Release.Namespace }}
+  labels:
+    {{- include "wandb.commonLabels" . | nindent 4 }}
+spec:
+  domains:
+  - {{ replace "https://" "" (replace "http://" "" .Values.global.host) }}
```

```suggestion
  - $defaultHost
```

What if there are multiple hosts? I think we support extra hosts